static/scripts/helpers/AlertMessageBuilder.js

```diff
@@ -0,0 +1,37 @@
+const moment = require('moment');
+
+moment.locale('de'); // set the localization
+
+function getIcon(status) {
+	switch (status) {
+		case 1:
+			return '<i class="fa fa-exclamation-circle text-danger"></i>';
+		case 2:
+			return '<i class="fa fa-check-circle text-success"></i>';
+		default:
+			return '';
+	}
+}
+
+export function loggedinMessageBuilder(message) {
```

Comment:
have you defined a message class that defines the structure? we do not need different builders when make html export dynamically based on objects data, so messages are build and when you want create a factory that returns them for loggedin oder not state